src/inputs/input.jsx

```diff
@@ -294,6 +356,21 @@ class Input extends React.PureComponent {
         return newType;
     }
 
+    setNewValue(value) {
+        const {
+            onChange,
+        } = this.props;
+
+        console.log('value', value);
```

Comment:
Please remove if the debugging is all done.

Reply:
Removing...

Reply:
Commit 7c8b691 fixes this.

Reply:
Looks good.  Resolving.